src/org/labkey/test/pages/core/login/LoginConfigurePage.java

```diff
@@ -36,6 +36,12 @@ public static LoginConfigurePage beginAt(WebDriverWrapper webDriverWrapper)
         return authenticationProvider.getNewDialog(getDriver());
     }
 
+    public boolean canAddConfiguration()
+    {
+        togglePrimaryConfiguration();
+        return new MultiMenu.MultiMenuFinder(getDriver()).withText("Add New Primary Configuration").findOptional(getDriver()).isPresent();
```

Comment:
Use the element cache version. With this technique, the button text could change and if the person doing the update didn't notice this method, the app admin check would become invalid because it would be validating the non-existence of a thing that doesn't ever exist.
```
try
{
    return elementCache().addPrimaryMenu.getComponentElement().isEnabled();
}
catch (NoSuchElementException nse)
{
    return false;
}
```

Reply:
I've refactored these such that the canAddConfiguration... methods use the same MenuFinder as the cached menu objects; that way we find the menus in regular testing and check for their non-presence with the same thing